test/runner/requirements/requirements.cloud.azure.txt

```diff
@@ -0,0 +1,17 @@
+packaging
```

Comment:
This file shouldn't be here. The integration test requirements for Azure come from `test/runner/requirements/integration.cloud.azure.txt`.